internal/core/src/mmap/Column.h

```diff
@@ -1,14 +1,18 @@
-// Copyright (C) 2019-2020 Zilliz. All rights reserved.
-//
-// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+// Licensed to the LF AI & Data foundation under one
```

Comment:
keep the files structure same with 2.2.0 branch, also the Column type is not for mmap only